### PR TITLE
proofread and minor changes to tabular

### PR DESCRIPTION
**Pull request by bradsaracik**

### CODE REVIEW

File: 09_tabular.ipynb [hunk 1]
```diff
@@ -40,14 +40,14 @@
...
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "> jargon: Continuous and categorical variables: \"Continuous variables\" are numerical data, such as \"age\" can be directly fed to the model, since you can add and multiply them directly. \"Categorical variables\" contain a number of discrete levels, such as \"movie id\", for which addition and multiplication don't have meaning (even if they're stored as numbers)."
+    "> jargon: Continuous and categorical variables: \"Continuous variables\" are numerical data and can be directly fed to the model, since you can add and multiply them, e.g. \"age\". \"Categorical variables\" contain a number of discrete levels, such as \"movie id\", for which addition and multiplication don't have meaning (even if they're stored as numbers)."
```
Comment (bradsaracik) [inline 09_tabular.ipynb]: Ordinal is used further down and it seemed to jar when I got there. Its nice to keep here about the 2 simple categories, but I also wonder if a new reader would be helped with that definition and something about things like North South East West vs compass headings, or categories like small medium large when they are here?

File: 09_tabular.ipynb [hunk 2]
```diff
@@ -629,9 +631,9 @@
...
     "\n",
     "- It returns the exact same object that's passed to it, after modifying the object *in-place*, and\n",
-    "- It runs the transform once, when data is first passed in, rather than lazily as the data is access.\n",
+    "- It runs the transform once, when data is first passed in, rather than lazily as the data is accessed.\n",
     "\n",
-    "`Categorify` is a `TabularProc` which replaces a column with a numeric categorical column. `FillMissing` is an `TabularProc` which replaces missing values with the median of the column, and creates a new boolean column that is set to True for any row where the value was missing. These two transforms are needed for nearly every tabular dataset you will use, so it's a good starting point for your data processing."
+    "`Categorify` is a `TabularProc` which replaces a column with a numeric categorical column. `FillMissing` is a `TabularProc` which replaces missing values with the median of the column, and creates a new boolean column that is set to True for any row where the value was missing. These two transforms are needed for nearly every tabular dataset you will use, so it's a good starting point for your data processing."
```
Comment (bradsaracik) [inline 09_tabular.ipynb]: I don't know if this comes up anywhere else - but 1) do we always use median? why / why not? 2) an example how is_missing helps / its intuition

File: 09_tabular.ipynb [hunk 3]
```diff
@@ -7367,7 +7369,7 @@
    "source": [
     "We can create a random forest just like we created a decision tree. Except now, we are also specifying parameters which indicate how many trees should be in the forest, how we should subset the data items (the rows), and how we should subset the fields (the columns).\n",
```
Comment (bradsaracik) [inline 09_tabular.ipynb]: the above is a nightly rather than stable. is this deliberate?

**Comment by bradsaracik** [comment]
line ~9000 under neural net theres a bit that says the competition uses MSE (metric is RMSE). I didnt come up with what I knew you'd like :)

File: 09_tabular.ipynb (hunk 2 above)
Comment (jph00) [reply 09_tabular.ipynb]: Basically the answer is - it doesn't matter. With the extra is_missing column, the tree can use it however it wants. I'll try to clarify

File: 09_tabular.ipynb (hunk 3 above)
Comment (jph00) [reply 09_tabular.ipynb]: Yes we need the latest version. It'll be out by the time the book is out.

**Comment by jph00** [comment]
What do you mean "I didnt come up with what I knew you'd like" ?

File: 09_tabular.ipynb (hunk 1 above)
Comment (jph00) [reply 09_tabular.ipynb]: Probably best to introduce things slowly

File: 09_tabular.ipynb [hunk 4]
```diff
@@ -9631,8 +9633,20 @@
    "display_name": "Python 3",
    "language": "python",
    "name": "python3"
+  },
```
Comment (jph00) [inline 09_tabular.ipynb]: Can you please install nbdev and run `nbdev_install_git_hooks` so you don't get this extra stuff in your PRs? It tends to cause conflicts.

**Comment by jph00** [comment]
Merged manually due to conflict.

**Comment by bradsaracik** [comment]
> What do you mean "I didnt come up with what I knew you'd like" ?

where you use tabular learner MSE, the draft states the competition was MSE, whereas it was RMSE stated throughout the chapter. Even though I understand the relationship, I wasn't comfortable I had a good succinct edit to clarify

**Pull request closed without merging**